cider/function/scalar/RuntimeFunctions.cpp

```diff
@@ -394,6 +394,23 @@ extern "C" ALWAYS_INLINE int8_t bit_is_set(const int64_t bitset,
              : 0;
 }
 
+extern "C" ALWAYS_INLINE int8_t bit_is_set_cider(const int64_t bitset,
```

Comment:
will this function be called in current test cases? and return bool would be better?

Reply:
done.